packages/experiment-browser/rollup.config.js

```diff
@@ -71,7 +71,7 @@ const configs = [
       entryFileNames: 'experiment.esm.js',
       format: 'esm',
     }),
-    external: ['@amplitude/ua-parser-js', '@amplitude/analytics-connector'],
+    external: ['@amplitude/analytics-connector'],
```

Comment:
shouldn't it stay an external here given that experiment-browser still depend on it ? bundling the ua-parser-js code would prevent deduplicating it with other packages using it, which hurts the "tree shakable" goal of that build.

Reply:
hey @stof, the thought here is that other packages that used to depend on it like , no longer do like @amplitude/analytics-connector, @amplitude/analytics-browser. our ultimate goal is to parse user agent server side so the SDKs no longer depend on the client side user agent parser. lmk if you have further thoughts!

Reply:
Anything that is in the `dependencies` section of the npm package should actually be listed in externals, so that they get imported from the installed dependency (allowing bundlers to dduplicate things).

Reply:
@stof gotcha! i see what you mean now. i realized that our esm build methods for analytics and experiment SDKs are different. i will open a pull request.